make/autoconf/libraries.m4

```diff
@@ -171,13 +171,11 @@ AC_DEFUN_ONCE([LIB_SETUP_LIBRARIES],
   LIB_SETUP_JVM_LIBS(TARGET)
 
   JDKLIB_LIBS="$BASIC_JDKLIB_LIBS"
-  JDKEXE_LIBS=""
```

Comment:
This is passed to LauncherCommon while JDKLIB\_LIBS is manually passed to every individual library it's used for. Why not unify them and pass JDKLIB\_LIBS to LibCommon instead? Other than that seems good

Reply:
JDKLIB_LIBS will be removed when I finish up the JDK_LIBS patch. (It is currently "-ljava -ljvm" on non-Windows, so basically a shortcut for setting dependencies on some commonly used JDK libraries.)

JDKEXE_LIBS was just dead code, so I removed it "eagerly" in this patch.